test/SlowTests/Issues/RavenDB-19625.cs

```diff
@@ -25,22 +28,75 @@ public void CanQueryIndexFilteredByDateTime()
 
             using (var session = store.OpenSession())
             {
-                session.Store(new Post { Id = "posts/1", Date = DateTime.UtcNow.AddMinutes(-5) });
+                session.Store(new Post {Id = "posts/1", Date = new DateTime(2023, 1, 1, 12, 11, 10)});
 
                 session.SaveChanges();
 
                 Indexes.WaitForIndexing(store);
-                
+
                 var res = session.Query<QueryDateTime_Index.Result, QueryDateTime_Index>()
                     .Where(x => x.Date < DateTime.UtcNow)
                     .ProjectInto<QueryDateTime_Index.Result>()
                     .ToList();
 
                 Assert.NotEmpty(res);
+                var hasTimeValues = GetDatabase(store.Database).Result.IndexStore.GetIndex(new QueryDateTime_Index().IndexName).IndexFieldsPersistence
+                    .HasTimeValues(nameof(QueryDateTime_Index.Result.Date));
+                Assert.True(hasTimeValues);
             }
         }
     }
 
+    [RavenFact(RavenTestCategory.Indexes)]
+    public void IndexBuiltBeforeJsDateIntroductionWillNotInsertTicks()
+    {
+        var backupPath = NewDataPath(forceCreateDir: true);
+        var fullBackupPath = Path.Combine(backupPath, "54_001_index_ver.ravendb-snapshot");
```

Comment:
Do we want to have test for v6.0 configuration too?

Reply:
Added in https://github.com/ravendb/ravendb/pull/18016